app/src/main/java/turtler/voyageur/fragments/CreateTripFragment.java

```diff
@@ -162,23 +163,8 @@ public void saveTrip() {
         if (friendsListIds.size() > 0) {
             for (int i = 0; i < friendsListIds.size(); i++) {
                 String friendId = friendsListIds.get(i);
-                newTrip.addTripFriend(ParseUser.createWithoutData(User.class, friendId));
-                ParseQuery friendQuery = new ParseQuery("User");
-                friendQuery.whereEqualTo("objectId", friendId);
-                friendQuery.findInBackground(new FindCallback() {
-                    @Override
-                    public void done(List objects, ParseException e) {
-                        if (objects != null && objects.size() > 0) {
-                            User friend = (User) objects.get(0);
-                            friend.addTrip(newTrip);
```

Comment:
this doesn't work. you need to be an authenticated user in order to update the user object